### PR TITLE
Installation and Deployment Tutorial

### PR DESCRIPTION
**Pull request by hthompson-a10**
## Description
Deployment tutorial

### CODE REVIEW

File: a10_octavia/doc/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,2035 @@
...
+
+*Note: Octavia verisons less than 4.1.1 have the `alembic_migrations` table instead*
+
+
+#### Step 5. Register and start a10-octavia services
+
+With `a10-octavia` installed, run the following command to register the services
+
+```shell
+install-a10-octavia
```
[the remark this answers is not in the document]
Comment (hthompson-a10) [reply a10_octavia/doc/tutorial.md:161]: This script is automatically added to the path when installing a10-octavia so it just becomes a command.

File: a10_octavia/doc/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,2035 @@
...
+ve13    Up    N/A   N/A    N/A   13   5254.0007.4a7b  10.0.13.10/24         1
+```
+
+
+##### A note on VLAN sharing
+
+
+##### Configuring for High Availability
+
+Now to throw high availability into the mix. If you haven't yet, please read over section #High-Availability section to review the basics of configuring a high availability deployment.
```
[the remark this answers is not in the document]
Comment (hthompson-a10) [reply a10_octavia/doc/tutorial.md:1327]: This is separate of that section

File: a10_octavia/doc/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,2035 @@
...
+#### Initial Setup
+
+There will be 2 Apache server instances running in the internal tenant network
+
+```
+openstack image create --disk-format qcow2 --container-format bare --public --file Apache_server1.qcow2 apache_server
+openstack flavor create --vcpu 1 --ram 2048 --disk 20 apache_flavor
+```
+
+Before continuing be sure to have completed the steps in the [#Installation] section
```
[the remark this answers is not in the document]
Comment (hthompson-a10) [reply a10_octavia/doc/tutorial.md:187]: I think this has been fixed. Looks like line numbers are off

File: a10_octavia/doc/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,2035 @@
...
+| monitor_port        | None                                 |
+| monitor_address     | None                                 |
+| backup              | False                                |
++---------------------+--------------------------------------+
+```
+
+##### Check the Thunder device
+
+```
+slb server 0b187895-9bd9-4a62-9a9a-8bed1a96eb6d 18.64.10.227
```
[the remark this answers is not in the document]
Comment (hthompson-a10) [reply a10_octavia/doc/tutorial.md:596]: This seems to have been fixed

File: a10_octavia/doc/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,2035 @@
...
+| Content types | None                                                                           |
+| Algorithm     | aes                                                                            |
+| Bit length    | 256                                                                            |
+| Secret type   | private                                                                        |
+| Mode          | cbc                                                                            |
+| Expiration    | None                                                                           |
++---------------+--------------------------------------------------------------------------------+
+```
+
+#### Step 4: Create a secrets container
```
[the remark this answers is not in the document]
Comment (hthompson-a10) [reply a10_octavia/doc/tutorial.md:1447]: Intended "This type of container holds any number of references to secrets." 

Stating "Create a secret container" would imply that the container itself is secret. Not that it stores secrets.

**Review by afrin-chakure-a10 — APPROVED** [review]
Looks good to me. 👍

File: a10_octavia/doc/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,2093 @@
...
+
+10. [Issues and Inquiries](#Issues-and-Inquiries)
+
+## Overview
+
+**This solution is currently in beta stage with limited support**
+
+The A10 Networks Octavia Driver allows for configuration of Thunder, vThunder, and AX Series Appliances deployed in
+an Openstack enviroment. While the default Octavia provider leverages an "Amphora per VIP" architecture,
+this provider driver uses a "Thunder per Tenant" architecture. Therefore, each tenant may only be serviced by a single
```
[the remark this answers is not in the document]
Comment (OmkarTelee-A10) [reply a10_octavia/doc/tutorial.md:39]: Whoever will make this change, needs to update this doc accordingly, approving for now

**Review by OmkarTelee-A10 — APPROVED** [review]

**Pull request merged**